app/src/main/java/com/github/se/gatherspot/model/MapViewModel.kt

```diff
@@ -125,15 +111,12 @@ class MapViewModel(application: Application) : AndroidViewModel(application) {
     registered_events =
         registered_events.union(list.map { EventFirebaseConnection().fetch(it) }).toMutableList()
 
-    Log.d(
-        "MapViewModel",
-        "fetchEvents: ${cameraPositionState.position.target.latitude}, ${cameraPositionState.position.target.longitude}")
     val list2 =
         EventFirebaseConnection()
             .fetchAllInPerimeter(
                 cameraPositionState.position.target.latitude,
                 cameraPositionState.position.target.longitude,
-                metersToDegree(cameraPositionState.position.target.latitude, 1000.0))
+                metersToDegree(1000.0, cameraPositionState.position.target.latitude))
```

Comment:
I guess that 1000 is the searching radius. But can be made explicit with the use of a val